core/src/main/java/com/klaytn/caver/tx/kct/KIP7Token.java

```diff
@@ -0,0 +1,776 @@
+
+    public static final String FUNC_ADDMINTER = "addMinter";
```

Comment:
Just curious. I think `FUNC_ADD_MINTER` is more readable. Can you explain why the function name is not separated?

Reply:
This code was generated by Caver-Java command line tool.
But I didn't modify this field name because i didn't think it was poorly readable.
If you think this field name is less readable, I'll edit it a s you suggested.

Reply:
@sirano11 No, you don't need to follow my suggestions all the time. If you think you're right, you just let me know your justification. In this case, it is auto-generated, just leave it as it is.

Or, If it is possible to auto-generate those variables as I suggested, then you can try it. (If you don't mind)

Reply:
Okay, I will leave this code as it is